src/lib/date.ts

```diff
@@ -262,6 +262,8 @@ export function formattedStartDateTime(startAt, endAt, timezone) {
     return `Starts ${singleDateTime(startAt, timezone)}`
   } else if (thisMoment.isBefore(endMoment)) {
     return `Ends ${singleDateTime(endAt, timezone)}`
+  } else if (endAt === null) {
```

Comment:
It might be clearer if we reordered this so we check for whether the current time is _after_ the `endMoment` and then if none of those cases match, return null. That way, this catch-all will also work if `startAt` is null, for instance.

i.e.
```javascript
} else if (thisMoment.isAfter(endMoment)) {
  return `Ended ${singleDate(endAt, timezone)}`
} else {
  return null
}

```